go/vt/vtctl/reparentutil/emergency_reparenter.go

```diff
@@ -211,13 +210,25 @@ func (erp *EmergencyReparenter) reparentShardLocked(ctx context.Context, ev *eve
 	// In case the user has specified a tablet specifically, then it is selected, as long as it is the most advanced.
 	// Here we also check for split brain scenarios and check that the selected replica must be more advanced than all the other valid candidates.
 	// We fail in case there is a split brain detected.
+	// The validCandidateTablets list is sorted by the replication positions with ties broken by promotion rules
 	intermediateSource, validCandidateTablets, err = erp.findMostAdvanced(validCandidates, tabletMap, opts)
 	if err != nil {
 		return err
 	}
 	erp.logger.Infof("intermediate source selected - %v", intermediateSource.Alias)
 
+	// After finding the intermediate source, we want to filter the valid candidate list by the following criterion -
```

Comment:
nit: you have multiple conditions, therefore "criterion" should be "criteria"

Reply:
no hablo inglés 😆

Reply:
Fixed 👍